src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -262,7 +262,10 @@
         public void WriteException(BinaryAnalyzerContext context, string errorMessage)
         {
             string fileHash = context.Hashes?.Sha256;
```

Comment:
>? [](http://example.com/codeflow?start=54&length=1)

nit: drop the ? and : so it will appear like this:

condition
    ? true
    : false

Reply:
For some reason, CodeFlow isn't allowing me to mark this one as resolved.